src/simparams.jl

```diff
@@ -76,6 +78,7 @@ function load_params(rng=MersenneTwister(0);
         population::DataFrame,
         infection_modulation_name::Union{Nothing,AbstractString}=nothing,
         infection_modulation_params::NamedTuple=NamedTuple{}(),
+        screening_params::Union{Nothing,ScreeningParams}=nothing,
```

Comment:
I think we could "hide" `screening_params` in `kwargs`, any reason why we are here initializing it explicitly to `nothing`?

Reply:
ok, i corrected it. probably it was remains after my previous version of the implementation